.github/workflows/ci-additional.yml

```
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        environment: ["mindeps-array-dataframe", "mindeps-bag-delayed"]
+        environment: ["mindeps-array-dataframe", "mindeps", "mindeps-distributed"]
```

Just a thought. This might make it more clear what the `mindeps` build is testing 

```suggestion
        environment: ["mindeps-array-dataframe", "mindeps-no-optional", "mindeps-distributed"]
```